deploy/docker/backend.Dockerfile

```
@@ -0,0 +1,14 @@
+FROM rust:1.79-alpine AS build
+RUN apk add --no-cache musl-dev pkgconfig openssl-dev
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)